third_party/java/arcs/build_defs/internal/kotlin.bzl

```diff
@@ -41,28 +41,54 @@ IS_BAZEL = not (hasattr(native, "genmpm"))
 
 # Kotlin Compiler Options
 KOTLINC_OPTS = [
+    "-Xallow-kotlin-package",
+    "-Xinline-classes",
     "-Xmulti-platform",
     "-Xuse-experimental=kotlin.ExperimentalMultiplatform",
 ]
 
+DISABLED_LINT_CHECKS = [
+    "PackageName",
+    "TopLevelName",
+]
+
+def _merge_lists(*lists):
+    result = {}
+    for l in lists:
+        for elem in l:
+            result[elem] = 1
+    return result.keys()
```

Comment:
No change needed, just want to gripe about how it's so silly that Starlark doesn't have "Set"s

Reply:
Tell me about it... :roll_eyes: